src/uu/split/split.md

```diff
@@ -11,3 +11,15 @@ Create output files containing consecutive or interleaved sections of input
 ## After Help
 
 Output fixed-size pieces of INPUT to PREFIXaa, PREFIXab, ...; default size is 1000, and default PREFIX is 'x'. With no INPUT, or when INPUT is -, read standard input.
+
+The SIZE argument is an integer and optional unit (example: 10K is 10*1024).
+Units are K,M,G,T,P,E,Z,Y,R,Q (powers of 1024) or KB,MB,... (powers of 1000).
+Binary prefixes can be used, too: KiB=K, MiB=M, and so on.
+
+CHUNKS may be:
+  N       split into N files based on size of input
+  K/N     output Kth of N to stdout
+  l/N     split into N files without splitting lines/records
+  l/K/N   output Kth of N to stdout without splitting lines/records
+  r/N     like 'l' but use round robin distribution
+  r/K/N   likewise but only output Kth of N to stdout
```

Comment:
yep, it would look better online with list, but I am afraid it will not look good in the terminal, i.e. it will look like this:

```
CHUNKS may be:

- N       split into N files based on size of input
- K/N     output Kth of N to stdout
- l/N     split into N files without splitting lines/records
- l/K/N   output Kth of N to stdout without splitting lines/records
- r/N     like 'l' but use round robin distribution
- r/K/N   likewise but only output Kth of N to stdout
```

or 

```
CHUNKS may be:

* N       split into N files based on size of input
* K/N     output Kth of N to stdout
* l/N     split into N files without splitting lines/records
* l/K/N   output Kth of N to stdout without splitting lines/records
* r/N     like 'l' but use round robin distribution
* r/K/N   likewise but only output Kth of N to stdout
```

which is not the same as in GNU version and could also be misleading/confusing - implying that CHUNKS should be entered with leading `-` or `*` followed by space and value or some such

Reply:
Hmm yeah, I understand your hesitation. I think we've done this before in some other utils too and it's generally fine to use the markdown syntax. Maybe some day we'll add some additional pre-processing for the help text :)

Reply:
sounds good. What would be the preference for the markdown list character then (to stay consistent with other utils)? Should it be `-` or `+` or `*` ?

Reply:
Let's see, we used `-` before (e.g. https://github.com/uutils/coreutils/pull/5260/files) so let's do that. Thanks!